src/components/hooks/usePerson.js

```diff
@@ -8,11 +8,12 @@ import {useState, useEffect} from 'react';
  * @returns {Person} Data of the person
  */
 export default function usePerson(personID, adapter) {
-  const [person, setPerson] = useState(undefined);
+  const [person, setPerson] = useState({});
```

Comment:
Why did you change this back?

Reply:
since we are not doing nested object, we dont have to validate the states anymore.